api/src/main/java/org/openmrs/module/isanteplusreports/library/cohort/ReportCohort.java

```diff
@@ -0,0 +1,23 @@
+package org.openmrs.module.isanteplusreports.library.cohort;
```

Comment:
@hmutaquiha please remove this whole class since it's meant to be used as an example